apps/web/src/components/Builders/Landing/Hero/index.tsx

```diff
@@ -42,13 +42,7 @@ export function Hero() {
             'after:absolute after:inset-x-0 after:bottom-0 after:z-10 after:h-24 after:bg-gradient-to-t after:from-black after:via-black/50 after:to-transparent',
           )}
         >
-          <div
-            className="h-full w-full bg-repeat max-md:bg-[length:200%]"
-            style={{ backgroundImage: `url(${heroBg.src})` }}
-          />
-          {/* <Image src={heroBg} alt="hero background" /> */}
-          {/* <GridHero columns={30} hasBlue className="hidden md:block" />
-          <GridHero columns={50} hasBlue className="md:hidden" /> */}
+          <GridHero hasBlue />
```

Comment:
could you make a bit more mobile-responsive by having a few of these with different cellSizes depending on screen size?